pkg/graph/schema.resolvers.go

```diff
@@ -1536,8 +1541,13 @@ func (r *queryResolver) Deployments(ctx context.Context, systemID string, deploy
 	return cedarDeployments, nil
 }
 
-func (r *queryResolver) Roles(ctx context.Context, systemID string, roleTypeID *string) ([]*models.CedarRole, error) {
-	cedarRoles, err := r.cedarCoreClient.GetRolesBySystem(ctx, systemID, null.StringFromPtr(roleTypeID))
+func (r *queryResolver) Roles(ctx context.Context, cedarSystemID string, roleTypeID *string) ([]*models.CedarRole, error) {
+	cedarSystem, err := r.cedarCoreClient.GetSystem(ctx, cedarSystemID)
```

Comment:
As we discussed in the meeting, calling `GetSystem()` to map `cedarSystemID` to a system's version ID should go in the CEDAR Core client.